packages/block-library/src/image/style.scss

```diff
@@ -1,6 +1,4 @@
 .wp-block-image {
-	max-width: 100%;
```

Comment:
Maybe this was meant for editor.css. Just noticed that if I give `.wp-block-image` a non-100% `max-width` for the front-end, that it will override this and break the wide and full images in the editor.

Reply:
It seems like we could remove this style, in my tests I'm not noticing different behavior in 2019 theme when this style is set. Any thoughts on this @jasmussen, @kjellr?

Reply:
Yeah, I think it's _generally_ safe to remove this. Things seem to work fine without it on my end — tested in Twenty Nineteen, Twenty Thirteen, and the Gutenberg Starter theme. 

It's true that if a theme were to set this to anything other than `max-width: 100%` in the editor, it'd break the editor's wide and full alignments. (@m-e-h, is that what you're referring to above?) But that would be the theme dev's decision to break things, so it'd be up to them to fix it. Everything seem to work fine if this style is removed out of the box.